src/package/nuspec/Microsoft.TestPlatform.TestHost.NetCore.targets

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Did we try CopyToPublishDirectory in the props itself?

Reply:
Tried. It works. Updated the PR